src/resources/locales/en.json

```diff
@@ -39,8 +39,10 @@
         "description": "Description",
         "emptyTagList": "You have no tagged ebook.",
         "entry": {
+            "continueAudiobooks": "Hör wieder auf",
             "continueReading": "Resume reading",
-            "lastAdditions": "Recently added"
+            "lastAdditions": "Recently added",
+            "lastAudiobooks": "letzte Hörbücher"
```

Comment:
Missing upper case letter in the first word, and second word should have lower case?